endpoints/openrtb2/auction_test.go

```diff
@@ -62,10 +62,6 @@ func TestJsonSampleRequests(t *testing.T) {
 			"Makes sure we handle (default) aliased bidders properly",
 			"aliased",
 		},
-		{
```

Comment:
May I ask what's the reason behind removing the malformed error test cases?

Reply:
Reverted